tests/test_dust.f90

```diff
@@ -0,0 +1,239 @@
+program test_dust
+   use precision_mod, only: fp
+   use Config_Opt_Mod, only: ConfigType
+   use ChemState_Mod, only: ChemStateType
+   use MetState_Mod, only: MetStateType
+   use DiagState_Mod, only: DiagStateType
```

Comment:
is this different than the StateMod? https://github.com/bbakernoaa/CATChem/blob/feature/catchem_begins/src/core/state_mod.F90

Reply:
Yeah similar to the state mod, but it wouldn't have declarations, and it would have more stuff in it. It would define how we want people to use CATChem, while still allowing us to move things around in the background.

Reply:
I think I'm just a little lost.  Happy to see what you have in mind though. It sounds good 👍